src/content/community/support/index.md

```diff
@@ -0,0 +1,77 @@
+
+Understanding the decentralized nature of Ethereum is vital because anyone claiming to be official support for Ethereum is probably trying to scam you! The best protection against scammers is educating yourself and taking security seriously.
+
+<DocLink to="/security/" title="Ethereum security and scam prevention" />
+
+<DocLink to="/learn/" title="Learn Ethereum fundamentals" />
+
+Despite the lack of official support, many groups, communities, and projects across the Ethereum ecosystem are happy to help.
+
+## Wallet support {#wallet-support}
+
+Having trouble with your wallet? Most wallets have dedicated support teams that can help.
+
+- [MetaMask](https://metamask.zendesk.com/hc/)
+- [Argent](https://support.argent.xyz/hc/)
+
+_This is not an exhaustive list. Need help finding support? Join the [ethereum.org discord](https://discord.gg/rZz26QWfCg) and we'll try to help._
+
+## Building dapps {#building-support}
+
+Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
+
+- [CryptoDevs discord](https://discord.gg/Z9TA39m8Yu)
+- [Ethereum StackExchange](https://ethereum.stackexchange.com/)
+- [StackOverflow](https://stackoverflow.com/questions/tagged/web3)
+
+### Tooling {#dapp-tooling}
+
+Does your question relate to a particular tool, project, or library? Most projects have chat servers or forums dedicated to supporting you.
+
+- [Solidity](https://gitter.im/ethereum/solidity/) <!-- What is the best place to send people to here? -->
+- [ethers.js](https://discord.gg/6jyGVDK6Jx)
+- [web3.js](https://discord.gg/GsABYQu4sC)
+- [Hardhat](https://discord.gg/xtrMGhmbfZ)
+- [Truffle](https://discord.gg/8uKcsccEYE)
+- [Chainlink](https://discord.gg/2YHSAey)
+- [Optimism](https://discord.gg/wGt8hjbZtu)
+
+## Running a node {#node-support}
+
+If you're running a node or validator, here are some communities that are dedicated to helping you get started.
+
+- [EthStaker discord](https://discord.io/ethstaker)
+- [EthStaker reddit](https://www.reddit.com/r/ethstaker)
+
+Most of the teams building Ethereum clients also have dedicated, public-facing, spaces where you can get support and ask questions.
+
+### Execution clients {#execution-clients}
+
+- [Geth](https://discord.gg/FqDzupGyYf)
+- [Nethermind](https://discord.gg/YJx3pm8z5C)
+- [Besu](https://discord.gg/p8djYngzKN)
+- [Ergion](https://github.com/ledgerwatch/erigon/issues) <!-- Does Erigon have a public facing channel? -->
+
+### Consensus clients {#consensus-clients}
+
+- [Prysm](https://discord.gg/prysmaticlabs)
+- [Nimbus](https://discord.gg/nSmEH3qgFv)
+- [Lighthouse](https://discord.gg/cyAszAh)
+- [Teku](https://discord.gg/7hPv2T6)
+- [Lodestar](https://discord.gg/aMxzVcr)
+
+## Common Questions
+
+- How do I....?
```

Comment:
Do you want a hand wtih some of these?

Reply:
[Added some initial questions.](https://github.com/ethereum/ethereum-org-website/pull/4521/commits/b0d947a23a7f2ed1e4f5344c3b4318a575189ca1) Feel free to suggest any others. I think with ~5ish questions we can get an 80/20 here without it being information overload.